src/yaml.rs

```diff
@@ -88,9 +88,7 @@ where
 	F: Copy + Fn(&mut String) -> Result<(), E>,
 {
 	match value {
-		serde_yaml::Value::Null => Ok(()),
-		serde_yaml::Value::Bool(_) => Ok(()),
-		serde_yaml::Value::Number(_) => Ok(()),
+		serde_yaml::Value::Null | serde_yaml::Value::Bool(_) | serde_yaml::Value::Number(_) => Ok(()),
```

Comment:
Was this `cargo fmt` or a manual edit? Because I really prefer the readability of the previous version :]